lib/providers/mixer.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+exports = module.exports = function (options) {
+	options = options || {};
+	var uri = options.uri || 'https://mixer.com/';
+	var user = options.uri ? options.uri + 'api/v1/users/current' : 'https://mixer.com/api/v1/users/current';
+
+	return {
+		protocol: 'oauth2',
+		auth: uri + 'oauth/authorize',
+		token: uri + 'api/v1/oauth/token',
+		scope: ['user:details:self'],
+		useParamsAuth: true,
+		scopeSeparator: ',',
```

Comment:
Not used by default, but this should be a space. We follow [the spec](https://tools.ietf.org/html/rfc6749#section-3.3) property :)